src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php

```diff
@@ -102,6 +102,6 @@ private function calculateTimeDelta(): int
 
         $serverTimestamp = (int) (string) $response->getContent();
```

Comment:
Can also remove these casts? There are not needed (maybe there are other ones to remove).

Reply:
too fast :smile: